admin/scripts/modules/uvod.php

```diff
@@ -93,7 +93,12 @@
 }
 
 if(post('zmenitUdaj')) {
-  dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+  $udaj = post('udaj');
+  if ($udaj['op' ?? null]) {
```

Comment:
Takto to bude vždy true, nechceš spíš `$udaj['op'] ?? null`? Můžeš a nemusíš si to zjednodušit i jako `post('udaj', 'op')`, to bude mít stejný efekt.

Mezery.